<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="font">
         <font>
          <family>Menlo</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionOpen"/>
    <addaction name="actionNew_File"/>
   </widget>
   <widget class="QMenu" name="menufunction">
    <property name="title">
     <string>Function</string>
    </property>
    <addaction name="actionPlot_Graph"/>
   </widget>
   <widget class="QMenu" name="menulanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionRussianLanguageSelect"/>
    <addaction name="actionEnglishLanguageSelect"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menufunction"/>
   <addaction name="menulanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPlot_Graph">
   <property name="text">
    <string>Plot Graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action> 
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRussianLanguageSelect">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEnglishLanguageSelect">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
